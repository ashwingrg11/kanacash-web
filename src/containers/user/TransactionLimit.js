import React, { Component } from "react";
import { Row, Col, Card, Accordion } from "react-bootstrap";
import PropTypes from "prop-types";

import InnerHTML from "dangerously-set-html-content";
import * as api from "../../services/axios/authApi";
import { connect } from "react-redux";
import { isEmpty } from "../../utils/isEmpty";

class TransactionLimit extends Component {
  constructor(props) {
    super(props);
    this.state = {
      widgetToken: "",
      key: "",
    };
  }

  componentDidMount = () => {
    this._mount = true;
    if (this._mount) {
      api
        .getWidgetToken()
        .then((res) => {
          this.setState({ widgetToken: res.data });
        })
        .catch((err) => {
          // console.log(err);
        });
      if (!isEmpty(this.props.user)) {
        if (this.props.user.current_tier === "LEVEL1") {
          this.setState({ key: "0" });
        } else if (this.props.user.current_tier === "LEVEL2") {
          this.setState({ key: "1" });
        } else {
          this.setState({ key: "2" });
        }
      }
    }
  };
  componentWillUnmount = () => {
    this._mount = false;
  };
  componentDidUpdate = (prevProps) => {
    if (prevProps.user !== this.props.user) {
      if (this.props.user.current_tier === "LEVEL1") {
        this.setState({ key: "0" });
      } else if (this.props.user.current_tier === "LEVEL2") {
        this.setState({ key: "1" });
      } else {
        this.setState({ key: "2" });
      }
    }
  };

  widget = () => {
    return `<div id="widget-root"></div>
      <script id="myScript"
      src="https://sandbox.api.machpay.com/v2/widget/widget.js"
      charset="utf-8"
      ></script>
      <script id="myScript">
      var widget = new MachnetWidget({
        elementId: 'widget-root',
        senderId: "${this.state.widgetToken.sender_id}",
        width: '100%',
        height: '400px',
        type: "tier",
        locale: 'en',
        multiStep: true,
        stylesheet: 'https://example.com/mystyle.css',
        token:
         "${this.state.widgetToken.token}"
      });
      widget.init();
      </script>`;
  };
  render() {
    return (
      <>
        <Row>
          <Col className="activity-title">Upgrade Limit</Col>
        </Row>
        <Row className="mt-3">
          <Col>
            <Card className="p-3">
              <Row>
                <Col>
                  All customers are categorized under three (3) transaction
                  limits based on the amount they can send. If you want to
                  transfer more funds your current transaction limit allows,
                  then you will need to a higher transaction limit. To upgrade,
                  you will be required to provide additional information.
                </Col>
              </Row>
            </Card>
          </Col>
        </Row>
        <Row className="mt-2">
          <Col lg="7">
            <InnerHTML html={this.widget()} />

            {/* <Card
              className="p-3"
              style={{
                backgroundColor: `${COLORS.LIGHT_BLUE}`,
                border: `1px solid ${COLORS.BLUE_COLOR}`,
              }}
            >
              <Row>
                <Col>
                  Your tier upgrade request is being processed. You will be
                  notified once approved. You can still create a transaction in
                  your current tier.
                </Col>
              </Row>
            </Card> */}
          </Col>
          <Col lg="5" className="mt-2 level-accordion">
            <Accordion activeKey={this.state.key}>
              <Card className="mb-2">
                <Accordion.Toggle
                  as={Card.Header}
                  eventKey="0"
                  onClick={() => this.setState({ key: "0" })}
                >
                  Level 1{" "}
                  {this.props.user.current_tier === "LEVEL1" && (
                    <span>(Current Limit)</span>
                  )}
                </Accordion.Toggle>
                <Accordion.Collapse eventKey="0">
                  <Card.Body>
                    <p>
                      <b>Requirement</b>
                    </p>
                    <ul className="mb-3">
                      <li>Full Name</li>
                      <li>Gender</li>
                      <li>Full Address (Street, City, State, Zip Code)</li>
                      <li>Date of Birth</li>
                      <li>Phone Number</li>
                      <li>OFAC and Sanction List Checks</li>
                    </ul>
                    <p>
                      <b>Note: </b>Limit for Level 1 customers
                    </p>
                    <ul>
                      <li> upto $500 per transaction</li>
                      <li>upto $500 per day</li>
                      <li>upto $500 per 15 days</li>
                      <li>upto $1,000 per 30 days</li>
                      <li>upto $3,000 per 6 months</li>
                    </ul>
                  </Card.Body>
                </Accordion.Collapse>
              </Card>
              <Card className="mb-2">
                <Accordion.Toggle
                  as={Card.Header}
                  eventKey="1"
                  onClick={() => this.setState({ key: "1" })}
                >
                  Level 2{" "}
                  {this.props.user.current_tier === "LEVEL2" && (
                    <span>(Current Limit)</span>
                  )}
                </Accordion.Toggle>
                <Accordion.Collapse eventKey="1">
                  <Card.Body>
                    <p>
                      <b>Requirement</b>
                    </p>
                    <ul className="mb-3">
                      <li>Id Number</li>
                      <li>ID issuing Authority</li>
                      <li>ID Expiry Date</li>
                      <li>Copy of Customer Id</li>
                    </ul>
                    <p>
                      <b>Note: </b>Limit for Level 2 customers
                    </p>
                    <ul>
                      <li> upto $1,000 per transaction</li>
                      <li>upto $2,999 per day</li>
                      <li>upto $2,999 per 15 days</li>
                      <li>upto $5,000 per 30 days</li>
                      <li>upto $9,999 per 6 months</li>
                    </ul>
                  </Card.Body>
                </Accordion.Collapse>
              </Card>
              <Card className="mb-2">
                <Accordion.Toggle
                  as={Card.Header}
                  eventKey="2"
                  onClick={() => this.setState({ key: "2" })}
                >
                  Level 3{" "}
                  {this.props.user.current_tier === "LEVEL3" && (
                    <span>(Current Limit)</span>
                  )}
                </Accordion.Toggle>
                <Accordion.Collapse eventKey="2">
                  <Card.Body>
                    <p>
                      <b>Requirement</b>
                    </p>
                    <ul className="mb-3">
                      <li> Full SSN</li>
                      <li>Occupation</li>
                      <li>Company Details</li>
                      <li>Purpose of Remittance</li>
                      <li>Sender Beneficiary Relationship</li>
                      <li>Source of Funds:</li>
                      <li>1. Bank Statement 2. Pay Slip</li>
                    </ul>
                    <p>
                      <b>Note: </b>Limit for Level 3 customers
                    </p>
                    <ul>
                      <li> upto $2,000 per transaction</li>
                      <li>upto $3,000 per day</li>
                      <li>upto $6,000 per 15 days</li>
                      <li>upto $10,000 per 30 days</li>
                      <li>upto $30,000 per 6 months</li>
                    </ul>
                  </Card.Body>
                </Accordion.Collapse>
              </Card>
            </Accordion>
          </Col>
        </Row>
      </>
    );
  }
}

// const mapDispatchToProps = {
// getActiveAccountPage,
// getCurrentUser,
// getActiveAccountSideBar,
// };

const mapStateToProps = (state) => ({
  user: state.auth.user,
});

TransactionLimit.propTypes = {
  user: PropTypes.object,
};
export default connect(mapStateToProps, null)(TransactionLimit);
