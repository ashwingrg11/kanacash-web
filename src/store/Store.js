import { createStore, combineReducers, applyMiddleware, compose } from "redux";
// import { composeWithDevTools } from 'redux-devtools-extension';
import Thunk from "redux-thunk";
import LoginReducer from "./reducers/Login";
import ForgotPasswordReducer from "./reducers/ForgotPassword";
import LoaderReducer from "./reducers/Loader";
import ModalReducer from "./reducers/Modal";
import NetworkReducer from "./reducers/Network";
import AuthReducer from "./reducers/Auth";
import ErrorReducer from "./reducers/Error";
import MiscellaneousReducer from "./reducers/MiscellaneousReducer";
import senderDetailReducers from "./reducers/SenderDetailReducer";
import TransactionReducer from "./reducers/TransactionReducer";
import SignUpReducer from "./reducers/SignUpReducer";
import AdminReducers from "./reducers/adminReducers";

const middleware = [Thunk];
const initialState = {};

const composeEnhancers =
  typeof window === "object" && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
    ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
        trace: true,
      })
    : compose;

const rootReducer = combineReducers({
  login: LoginReducer,
  forgotPassword: ForgotPasswordReducer,
  loader: LoaderReducer,
  modal: ModalReducer,
  network: NetworkReducer,
  auth: AuthReducer,
  error: ErrorReducer,
  miscellaneous: MiscellaneousReducer,
  senderDetail: senderDetailReducers,
  transaction: TransactionReducer,
  signup: SignUpReducer,
  admin: AdminReducers,
});

const store = createStore(
  rootReducer,
  initialState,
  composeEnhancers(applyMiddleware(...middleware))
);
export default store;
