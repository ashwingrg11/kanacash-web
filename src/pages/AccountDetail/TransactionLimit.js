import React, { Component } from "react";
import { Container, Row, Col } from "react-bootstrap";

import MyAccountSidebar from "../../components/MyAccountSidebar";
import { withRouter } from "react-router-dom";
import Header from "../../containers/layout/Header";
import { getBeneficiaries } from "../../store/actions/senderDetail";
import { connect } from "react-redux";
import Footer from "../../containers/layout/Footer";
import AccountHeader from "../../containers/user/AccountHeader";
import TransactionLimit from "../../containers/user/TransactionLimit";
import Disclaimer from "../../components/Disclaimer/Disclaimer";
class AccountDetail extends Component {
  render() {
    return (
      <>
        <Header />
        <div className="shadow-line"></div>
        <section className="admin-wrapper">
          <Container>
            <Row>
              <Col>
                <AccountHeader />
                <Row className="admin-content">
                  <Col xl="3" lg="4" md="5" sm="12">
                    <MyAccountSidebar />
                  </Col>
                  <Col xl="9" lg="8" md="7" sm="12">
                    <TransactionLimit />
                  </Col>
                </Row>
              </Col>
            </Row>
          </Container>
        </section>
        <Footer />
        {!localStorage.acceptPolicy && <Disclaimer />}
      </>
    );
  }
}
const mapStateToProps = (state) => ({
  beneficiary: state.senderDetail.beneficiaries,
  activePage: state.senderDetail.activePage,
  acceptPolicy: state.auth.accept_policy,
});
const mapActionToProps = { getBeneficiaries };

export default connect(
  mapStateToProps,
  mapActionToProps
)(withRouter(AccountDetail));
