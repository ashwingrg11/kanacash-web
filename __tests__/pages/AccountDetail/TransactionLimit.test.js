import React from "react";
import TransactionLimit from "../../../src/pages/AccountDetail/TransactionLimit";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import Store from "../../../src/store/Store";

import renderer from "react-test-renderer";
describe("<TransactionLimit />", () => {
  it("TransactionLimit renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={Store}>
          <Router>
            <TransactionLimit />
          </Router>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
