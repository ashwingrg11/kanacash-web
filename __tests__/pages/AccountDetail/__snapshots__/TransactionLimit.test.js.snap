// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionLimit /> TransactionLimit renders correctly 1`] = `
Array [
  <header
    className="header-main"
  >
    <div
      className="container"
    >
      <nav
        className="navbar navbar-expand-lg navbar-light bg-light"
      >
        <div
          className="navbar-brand"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          <img
            alt="logo"
            className="logo"
            src=""
          />
        </div>
        <button
          aria-controls="basic-navbar-nav"
          aria-label="Toggle navigation"
          className="navbar-toggler collapsed"
          onClick={[Function]}
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </button>
        <div
          aria-expanded={null}
          className="navbar-collapse collapse"
          id="basic-navbar-nav"
        >
          <div
            className="ml-md-auto navbar-nav"
            onKeyDown={[Function]}
          >
            <span
              className="btn btn-primary btn-theme btn-primary-border"
              onClick={[Function]}
            >
              Sign in
            </span>
            <button
              className="btn btn-primary btn-theme btn-primary-theme"
              onClick={[Function]}
            >
              Sign up
            </button>
          </div>
        </div>
      </nav>
    </div>
  </header>,
  <div
    className="shadow-line"
  />,
  <section
    className="admin-wrapper"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col"
        >
          <h2
            className="account-title"
          >
            My Account
          </h2>
          <p>
            Everything we know about you.
          </p>
          <div
            className="admin-content row"
          >
            <div
              className="col-xl-3 col-lg-4 col-md-5 col-sm-12"
            >
              <ul
                className="sidebar-menu"
              >
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#1077BE",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="activeImg"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Profile
                  </div>
                </li>
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#4F4F4F",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="img"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Beneficiary
                  </div>
                </li>
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#4F4F4F",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="img"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Bank Accounts
                  </div>
                </li>
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#4F4F4F",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="img"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Debit Cards
                  </div>
                </li>
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#4F4F4F",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="img"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Transactions
                  </div>
                </li>
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#4F4F4F",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="img"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Transaction Limit
                  </div>
                </li>
                <li>
                  <div
                    className="d-flex align-items-center"
                    onClick={[Function]}
                    style={
                      Object {
                        "color": "#4F4F4F",
                        "cursor": "pointer",
                        "fontSize": "16px",
                        "fontWeight": "bold",
                        "marginBottom": "10px",
                      }
                    }
                  >
                     
                    <img
                      alt="img"
                      src=""
                      style={
                        Object {
                          "marginRight": "10px",
                          "paddingBottom": "3px",
                        }
                      }
                    />
                    Settings
                  </div>
                </li>
              </ul>
            </div>
            <div
              className="col-xl-9 col-lg-8 col-md-7 col-sm-12"
            >
              <div
                className="row"
              >
                <div
                  className="activity-title col"
                >
                  Upgrade Limit
                </div>
              </div>
              <div
                className="mt-3 row"
              >
                <div
                  className="col"
                >
                  <div
                    className="p-3 card"
                  >
                    <div
                      className="row"
                    >
                      <div
                        className="col"
                      >
                        All customers are categorized under three (3) transaction limits based on the amount they can send. If you want to transfer more funds your current transaction limit allows, then you will need to a higher transaction limit. To upgrade, you will be required to provide additional information.
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="mt-2 row"
              >
                <div
                  className="col-lg-7"
                >
                  <div />
                </div>
                <div
                  className="mt-2 level-accordion col-lg-5"
                >
                  <div
                    className="accordion"
                  >
                    <div
                      className="mb-2 card"
                    >
                      <div
                        className="card-header"
                        onClick={[Function]}
                      >
                        Level 1
                         
                      </div>
                      <div
                        aria-expanded={null}
                        className="collapse"
                      >
                        <div
                          className="card-body"
                        >
                          <p>
                            <b>
                              Requirement
                            </b>
                          </p>
                          <ul
                            className="mb-3"
                          >
                            <li>
                              Full Name
                            </li>
                            <li>
                              Gender
                            </li>
                            <li>
                              Full Address (Street, City, State, Zip Code)
                            </li>
                            <li>
                              Date of Birth
                            </li>
                            <li>
                              Phone Number
                            </li>
                            <li>
                              OFAC and Sanction List Checks
                            </li>
                          </ul>
                          <p>
                            <b>
                              Note: 
                            </b>
                            Limit for Level 1 customers
                          </p>
                          <ul>
                            <li>
                               upto $500 per transaction
                            </li>
                            <li>
                              upto $500 per day
                            </li>
                            <li>
                              upto $500 per 15 days
                            </li>
                            <li>
                              upto $1,000 per 30 days
                            </li>
                            <li>
                              upto $3,000 per 6 months
                            </li>
                          </ul>
                        </div>
                      </div>
                    </div>
                    <div
                      className="mb-2 card"
                    >
                      <div
                        className="card-header"
                        onClick={[Function]}
                      >
                        Level 2
                         
                      </div>
                      <div
                        aria-expanded={null}
                        className="collapse"
                      >
                        <div
                          className="card-body"
                        >
                          <p>
                            <b>
                              Requirement
                            </b>
                          </p>
                          <ul
                            className="mb-3"
                          >
                            <li>
                              Id Number
                            </li>
                            <li>
                              ID issuing Authority
                            </li>
                            <li>
                              ID Expiry Date
                            </li>
                            <li>
                              Copy of Customer Id
                            </li>
                          </ul>
                          <p>
                            <b>
                              Note: 
                            </b>
                            Limit for Level 2 customers
                          </p>
                          <ul>
                            <li>
                               upto $1,000 per transaction
                            </li>
                            <li>
                              upto $2,999 per day
                            </li>
                            <li>
                              upto $2,999 per 15 days
                            </li>
                            <li>
                              upto $5,000 per 30 days
                            </li>
                            <li>
                              upto $9,999 per 6 months
                            </li>
                          </ul>
                        </div>
                      </div>
                    </div>
                    <div
                      className="mb-2 card"
                    >
                      <div
                        className="card-header"
                        onClick={[Function]}
                      >
                        Level 3
                         
                      </div>
                      <div
                        aria-expanded={null}
                        className="collapse"
                      >
                        <div
                          className="card-body"
                        >
                          <p>
                            <b>
                              Requirement
                            </b>
                          </p>
                          <ul
                            className="mb-3"
                          >
                            <li>
                               Full SSN
                            </li>
                            <li>
                              Occupation
                            </li>
                            <li>
                              Company Details
                            </li>
                            <li>
                              Purpose of Remittance
                            </li>
                            <li>
                              Sender Beneficiary Relationship
                            </li>
                            <li>
                              Source of Funds:
                            </li>
                            <li>
                              1. Bank Statement 2. Pay Slip
                            </li>
                          </ul>
                          <p>
                            <b>
                              Note: 
                            </b>
                            Limit for Level 3 customers
                          </p>
                          <ul>
                            <li>
                               upto $2,000 per transaction
                            </li>
                            <li>
                              upto $3,000 per day
                            </li>
                            <li>
                              upto $6,000 per 15 days
                            </li>
                            <li>
                              upto $10,000 per 30 days
                            </li>
                            <li>
                              upto $30,000 per 6 months
                            </li>
                          </ul>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <footer
    className="footer-section text-center"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-12"
        >
          <div
            className="foot-nav"
          >
            <a
              aria-current="page"
              className="link active"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
            <a
              aria-current={null}
              className="link"
              href="/about-us"
              onClick={[Function]}
            >
              About Us
            </a>
            <a
              aria-current={null}
              className="link"
              href="/terms-of-service"
              onClick={[Function]}
            >
              Terms
            </a>
            <a
              aria-current={null}
              className="link"
              href="/privacy-policy"
              onClick={[Function]}
            >
              Privacy Policy
            </a>
            <a
              aria-current={null}
              className="link"
              href="/copyright"
              onClick={[Function]}
            >
              Copyright
            </a>
            <a
              aria-current={null}
              className="link"
              href="/contact-us"
              onClick={[Function]}
            >
              Contact
            </a>
          </div>
        </div>
        <div
          className="copyright col-sm-12"
        >
          <p>
            © Golden Money Transfer Inc. 2020. All Rights Reserved.
             
            <b>
              Licensed by
            </b>
             Department of Financial Institutions, Nevada
          </p>
        </div>
      </div>
    </div>
  </footer>,
  <div
    className="disclaimer text-center"
  >
    <div
      className="container"
    >
      <div
        className="policy"
      >
         
        This website uses cookies to ensure you get the best experience on our website, as described in our
         
        <span
          className="btn-link"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
           
            privacy policy.
        </span>
      </div>
      <div
        className="policy1"
      >
        This website uses cookies,
        <span
          className="btn-link"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
           
            learn more.
        </span>
      </div>
      <div
        className="buttonDiv"
      >
        <button
          className="btn-outline-warning"
          onClick={[Function]}
          style={
            Object {
              "border": "none",
              "borderRadius": "50px",
              "color": "black",
              "padding": "5px 15px",
            }
          }
        >
          Accept
        </button>
      </div>
    </div>
  </div>,
]
`;
